Vagrantfile

```diff
@@ -12,7 +12,7 @@ Vagrant.configure("2") do |config|
 
   # Every Vagrant development environment requires a box. You can search for
   # boxes at https://vagrantcloud.com/search.
-  config.vm.box = "debian/contrib-jessie64"
+  config.vm.box = "generic/ubuntu1804"
```

Comment:
This VM is almost 4 times bigger than the other one. Is there anything lighter?

Reply:
There's generic/alpine38 if we wanted to get light - ubuntu is pretty bloated. 

```
/tmp/foo/ du -ms ~/.vagrant.d/boxes/*
546	         /Users/b/.vagrant.d/boxes/generic-VAGRANTSLASH-alpine38
2940	/Users/b/.vagrant.d/boxes/generic-VAGRANTSLASH-ubuntu1804
```